_posts/2016/2016-03-17-team-orange-mallorca.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Developer team enjoys an innovative and productive offsite week on Mallorca"
```

Comment:
I'm sorry for this, but the title is too long.
We'd like it short and simple.
Suggestion: Productive team offsite on Mallorca